pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -2778,6 +2785,13 @@ func TestGetNodesToDaemonPods(t *testing.T) {
 			newPod("non-matching-owned-0-", "node-0", simpleDaemonSetLabel2, ds),
 			newPod("non-matching-orphan-1-", "node-1", simpleDaemonSetLabel2, nil),
 			newPod("matching-owned-by-other-0-", "node-0", simpleDaemonSetLabel, ds2),
+			func() *v1.Pod {
```

Comment:
nit: I would suggest adding another `succeeded` pod for symmetry.

Reply:
added